src/ontology/mondo-edit.obo

```diff
@@ -366748,16 +366748,26 @@ relationship: has_characteristic MONDO:0021136 {source="MONDO:0015906"} ! rare
 [Term]
 id: MONDO:0016819
 name: Moebius syndrome-axonal neuropathy-hypogonadotropic hypogonadism syndrome
-def: "This syndrome is characterized by the association of MC6bius syndrome (congenital facial palsy with impaired ocular abduction) with peripheral axonal neuropathy and hypogonadotropic hypogonadism." [Orphanet:2560]
+def: "This syndrome is characterized by the association of Moebius syndrome (congenital facial palsy with impaired ocular abduction) with peripheral axonal neuropathy and hypogonadotropic hypogonadism." [Orphanet:2560]
 subset: gard_rare {source="GARD:3698"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_malformation_syndrome {source="Orphanet:2560"}
 subset: orphanet_rare {source="Orphanet:2560"}
 subset: rare
+synonym: "Moebius axonal neuropathy hypogonadism" EXACT [MONDO:0023820]
+synonym: "Moebius syndrome with hypogonadotrophic hypogonadism and progressive peripheral neuropathy axonal and demyelinating type" RELATED [GARD:0003698]
 xref: GARD:3698 {source="Orphanet:2560"}
+xref: MESH:C535806 {source="MONDO:equivalentTo"}
 xref: Orphanet:2560 {source="MONDO:equivalentTo"}
 xref: UMLS:C2931024 {source="Orphanet:2560/e", source="MONDO:equivalentTo", source="Orphanet:2560"}
+xref: UMLS:C2931024 {source="GARD:0003698", source="MONDO:relatedTo"}
```

Comment:
These two xrefs to `UMLS:C2931024` should be combined.

Reply:
yes, I merged these by running make norm. You'll see a few other edits on this PR b/c of the normalize. Thanks!